core/src/main/java/org/elasticsearch/action/get/MultiGetRequest.java

```diff
@@ -493,8 +525,8 @@ public static void parseDocuments(XContentParser parser, List<Item> items) throw
     }
```

Comment:
Where do we need `public static void parseDocuments(XContentParser parser, List<Item> items)` in the code? I think we can remove this and only leave "add(...)" as an entry point to parsing.

Reply:
The `add(...)` method is pretty large already, I like not to make it larger than it already is by adding the that is now in `parseDocuments(...)` to it.